src/lib.rs

```diff
@@ -138,6 +138,21 @@ struct Registers {
     _reserved13: [u8; 3],
 }
 
+/// PL011 Errors
+#[derive(Debug, PartialEq)]
```

Comment:
Derive `Copy`, `Clone` and `Eq` as well.

Reply:
Done